client/extensions/woocommerce/app/settings/shipping/shipping-methods/free-shipping.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormCurrencyInput from 'components/forms/form-currency-input';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+class FreeShippingMethod extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			everyone: props.everyone,
+			currencySymbolPrefix: '$',
+			currencySymbolSuffix: ''
+		};
+	}
+
+	renderMinSpendBox() {
+		return (
+			<FormCurrencyInput
+				currencySymbolPrefix={ this.state.currencySymbolPrefix }
+				currencySymbolSuffix={ this.state.currencySymbolSuffix }
+				value={ this.state.minSpend } />
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		const onOptionChange = ( event ) => {
+			this.setState( { everyone: 'everyone' === event.currentTarget.value } );
+		};
+
+		return (
+			<div className="shipping-methods__method-container">
+				<FormLabel>{ translate( 'Who gets free shipping?' ) }</FormLabel>
+				<div className="shipping-methods__free-shipping-option">
+					<FormRadio value="everyone"
+						checked={ this.state.everyone }
+						onChange={ onOptionChange } />
+					{ translate( 'Everyone!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Everyone', { context: 'Filter label for posts list'} )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).